### PR TITLE
Fix issue in UI, BE linkage issue and webpack issues

### PR DESCRIPTION
**Pull request by ManiEdcast**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,11 +60,29 @@
...
     "@babel/preset-env": "^7.19.3",
+    "@babel/preset-react": "^7.18.6",
     "babel-loader": "^8.2.5",
+    "brotli-webpack-plugin": "^1.1.0",
+    "compression-webpack-plugin": "^10.0.0",
+    "cross-env": "^7.0.3",
+    "css-loader": "^0.28.10",
+    "del": "^7.0.0",
+    "esbuild-loader": "^2.20.0",
+    "file-loader": "^6.2.0",
```
Comment (prachikarekar) [inline package.json:75]: this doesn't seem to be used in the below code, can we remove this?

File: package.json [hunk 2]
```diff
@@ -62,11 +60,29 @@
     ]
   },
   "devDependencies": {
+    "@babel/cli": "^7.19.3",
```
Comment (prachikarekar) [inline package.json:63]: this doesn't seem to be used in the below code, can we remove this?

File: package.json [hunk 3]
```diff
@@ -62,11 +60,29 @@
...
     "@babel/core": "^7.19.3",
+    "@babel/node": "^7.19.1",
```
Comment (prachikarekar) [inline package.json:65]: this doesn't seem to be used in the below code, can we remove this?

File: package.json [hunk 4]
```diff
@@ -62,11 +60,29 @@
...
     "@babel/preset-env": "^7.19.3",
+    "@babel/preset-react": "^7.18.6",
     "babel-loader": "^8.2.5",
+    "brotli-webpack-plugin": "^1.1.0",
+    "compression-webpack-plugin": "^10.0.0",
```
Comment (prachikarekar) [inline package.json:70]: this doesn't seem to be used in the below code, can we remove this?

File: package.json [hunk 5]
```diff
@@ -62,11 +60,29 @@
...
+    "cross-env": "^7.0.3",
+    "css-loader": "^0.28.10",
```
Comment (prachikarekar) [inline package.json:72]: this doesn't seem to be used in the below code, can we remove this?

File: package.json [hunk 6]
```diff
@@ -62,11 +60,29 @@
...
+    "del": "^7.0.0",
```
Comment (prachikarekar) [inline package.json:73]: this doesn't seem to be used in the below code, can we remove this?

File: package.json [hunk 7]
```diff
@@ -62,11 +60,29 @@
...
+    "esbuild-loader": "^2.20.0",
+    "file-loader": "^6.2.0",
+    "hbs": "^4.0.0",
+    "mini-css-extract-plugin": "^2.6.1",
+    "postcss-loader": "^7.0.1",
     "redux": "^4.2.0",
+    "sass-loader": "^13.1.0",
```
Comment (prachikarekar) [inline package.json:80]: this doesn't seem to be used in the below code, can we remove this?

File: package.json [hunk 8]
```diff
@@ -62,11 +60,29 @@
...
+    "speed-measure-webpack-plugin": "^1.5.0",
+    "style-loader": "^0.19.1",
```
Comment (prachikarekar) [inline package.json:82]: this doesn't seem to be used in the below code, can we remove this?

File: package.json [hunk 9]
```diff
@@ -62,11 +60,29 @@
...
     "webpack": "^5.74.0",
-    "webpack-cli": "^4.10.0"
+    "webpack-bundle-analyzer": "^4.6.1",
```
Comment (prachikarekar) [inline package.json:84]: this doesn't seem to be used in the below code, can we remove this?

File: package.json [hunk 10]
```diff
@@ -62,11 +60,29 @@
...
+    "webpack-cli": "^4.10.0",
```
Comment (prachikarekar) [inline package.json:85]: can we remove this?

File: server/config/router.js [hunk 11]
```diff
@@ -17,7 +17,7 @@
  
  
    // read only once (version changes only on new build)
-   var version = require(path.resolve('.') + '/version.json');
+  //  var version = require(path.resolve('.') + '/version.json');
```
Comment (prachikarekar) [inline server/config/router.js:20]: can we keep this line for now?

File: server/config/view.js [hunk 12]
```diff
@@ -1,54 +1,47 @@
-// do not use es6 syntax in this file as it doesn't handled by babel
-window._webConstants = {{{env_vars}}};
```
Comment (prachikarekar) [inline server/config/view.js:2]: we need this line

Comment (ManiEdcast) [reply server/config/view.js:2]: May be we should add it from somewhere else, this file contain one exportable module based code. Cant declare this inside it.

File: package.json (hunk 10 above)
Comment (ManiEdcast) [reply package.json:85]: It is required for production webpack configuration

File: package.json (hunk 9 above)
Comment (ManiEdcast) [reply package.json:84]: It is required for production webpack configuration

File: package.json [hunk 13]
```diff
@@ -62,11 +60,29 @@
...
+    "css-loader": "^0.28.10",
+    "del": "^7.0.0",
+    "esbuild-loader": "^2.20.0",
+    "file-loader": "^6.2.0",
+    "hbs": "^4.0.0",
+    "mini-css-extract-plugin": "^2.6.1",
+    "postcss-loader": "^7.0.1",
     "redux": "^4.2.0",
+    "sass-loader": "^13.1.0",
+    "speed-measure-webpack-plugin": "^1.5.0",
```
[the remark this answers is not in the document]
Comment (ManiEdcast) [reply package.json:81]: It is required for production webpack configuration

File: package.json (hunk 5 above)
Comment (ManiEdcast) [reply package.json:72]: It is required for production webpack configuration

File: package.json (hunk 4 above)
Comment (ManiEdcast) [reply package.json:70]: It is required for production webpack configuration

File: package.json [hunk 14]
```diff
@@ -62,11 +60,29 @@
     ]
   },
   "devDependencies": {
+    "@babel/cli": "^7.19.3",
     "@babel/core": "^7.19.3",
+    "@babel/node": "^7.19.1",
     "@babel/preset-env": "^7.19.3",
+    "@babel/preset-react": "^7.18.6",
     "babel-loader": "^8.2.5",
+    "brotli-webpack-plugin": "^1.1.0",
```
[the remark this answers is not in the document]
Comment (ManiEdcast) [reply package.json:69]: It is required for production webpack configuration

File: package.json (hunk 6 above)
Comment (ManiEdcast) [reply package.json]: del package removed, others need.

File: package.json (hunk 3 above)
Comment (ManiEdcast) [reply package.json]: removed

File: server/config/router.js (hunk 11 above)
Comment (ManiEdcast) [reply server/config/router.js]: Done

File: package.json (hunk 2 above)
Comment (ManiEdcast) [reply package.json]: removed

File: package.json (hunk 1 above)
Comment (ManiEdcast) [reply package.json:72]: del package removed

File: server/config/view.js (hunk 12 above)
Comment (prachikarekar) [reply server/config/view.js:2]: we will explore this point again

**Pull request merged**